### ISSUE TITLE
Incompatibility with PopOut Resizer

### ISSUE THREAD

**Issue by nulmas** [sPOiDar/fvtt-module-stream-view#16]
If PopOut Resizer is active, the chat window will be placed near the middle of the screen and have the wrong size. Using the settings to change its position will also have no effect.

I haven't tried it with any other windows (like the combat  tracker), but I suspect the behavior may be similar.
![image](https://user-images.githubusercontent.com/79886727/153726147-e7034c8d-102a-4441-9bd8-975bf219d2b9.png)

**Comment by sPOiDar** [comment]
I don't believe there's anything sensible I can do fix this from my end. I'll leave a note for Cardagon on how to avoid messing with the stream user's popouts, in case they're willing to add compatibility.
